src/Validation/Rule/LoneAnonymousOperationRule.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Digia\GraphQL\Validation\Rule;
+
+use Digia\GraphQL\Error\ValidationException;
+use Digia\GraphQL\Language\AST\Node\DocumentNode;
+use Digia\GraphQL\Language\AST\Node\NodeInterface;
+use Digia\GraphQL\Language\AST\Node\OperationDefinitionNode;
+use function Digia\GraphQL\Validation\anonymousOperationNotAloneMessage;
+
+/**
+ * Lone anonymous operation
+ *
+ * A GraphQL document is only valid if when it contains an anonymous operation
```

Comment:
Typo: "if when"

Reply:
Typo was taken directly from the reference impl. :D